video/cloud-client/analyze/beta_snippets_test.py

```diff
@@ -110,12 +122,12 @@ def test_annotation_to_storage_streaming(capsys, video_path, bucket):
             break
 
     # Confirm that one output blob had been written to GCS.
-    assert len(blobs) == 1
+    assert len(blobs) >= 0
```

Comment:
This will make the test always pass

Reply:
It should be > 0, right?

Reply:
Welp pay attention to typos. Changing to >0.

Reply:
Yes? idk I didn't pay too close attention to it.